docs/products/kafka/kafka-connect/howto/mqtt-sink-connector.rst

```diff
@@ -0,0 +1,97 @@
+Create a MQTT sink connector
+==============================
+
+The `MQTT sink connector <https://docs.lenses.io/5.0/integrations/connectors/stream-reactor/sinks/mqttsinkconnector/>`_ copies messages from Apache Kafka® topic to a MQTT queue. 
+
+.. note::
+
+    You can check the full set of available parameters and configuration options in the `connector's documentation <https://docs.lenses.io/5.0/integrations/connectors/stream-reactor/sinks/mqttsinkconnector/>`_.
+
+.. Tip::
+
+    The connector can be used to sink messages to RabbitMQ® where `RabbitMQ MQTT plugin <https://www.rabbitmq.com/mqtt.html>`_ is enabled. 
+
+
+.. _connect_mqtt_rbmq_sink_prereq:
+
+Prerequisites
+-------------
+
+To set up a MQTT sink connector, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
+
+.. Tip::
+
+  The connector will write to a topic defined in the ``"connect.mqtt.kcql"`` configuration, so either create the topic in your Kafka service, or enable the ``auto_create_topic`` parameter so that the topic will be created automatically.
+
+Furthermore you need to collect the following information about the sink MQTT server upfront:
+
+* ``USERNAME``: The MQTT username to connect
+* ``PASSWORD``: The password for the username selected
+* ``HOST``: The MQTT hostname
+* ``PORT``: MQTT port (usually 1883)
+* ``KCQL_STATEMENT``: The KCQL statement to be used in the following format::
+
+    INSERT INTO <mqtt-topic>
+    SELECT FIELD, ... 
+    FROM <kafka-topic>
```

Comment:
This is a multi-line string (fair enough), but further down you suggest putting it into JSON, which doesn't support multi-line strings. So should the example here say:
```
The KCQL statement to be used, as a "single line" string without linebreaks::

    INSERT INTO <mqtt-topic> SELECT FIELD ... FROM <kafka-topic>
```